src/screens/OtherProfile/index.js

```diff
@@ -289,8 +298,8 @@ const OtherProfile = () => {
         <React.Fragment>
           {__renderFollowingButton()}
           <GlobalButton onPress={onCreateChat}>
-            <View style={styles.btnMsg}>
-              <EnveloveBlueIcon width={20} height={20} fill={colors.bondi_blue} />
+            <View style={styles.btnMsg(isAnonimity)}>
+              <EnvelopeIcon color={isAnonimity ? COLORS.anon_primary : COLORS.signed_primary} />
             </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of the `EnveloveBlueIcon` with the new `EnvelopeIcon` component is correctly done. However, as mentioned earlier, it would be good to ensure that the `EnvelopeIcon` component is properly tested and its usage here does not introduce any regressions.



<!-- This is an auto-generated comment by OSS CodeRabbit -->